docs/reverse_proxy.md

```diff
@@ -160,6 +161,52 @@ backend matrix
   server matrix 127.0.0.1:8008
 ```
 
+### Relayd
+
+```
+table <webserver>    { 127.0.0.1 }
+table <matrixserver> { 127.0.0.1 }
+
+http protocol "https" {
+    tls { no tlsv1.0, ciphers "HIGH" }
+    tls keypair "example.com"
+    match header set "X-Forwarded-For"   value "$REMOTE_ADDR"
+    match header set "X-Forwarded-Proto" value "https"
```

Comment:
We [recently discovered](https://github.com/matrix-org/synapse/pull/9512) that nginx modified pass the HTTP `Host` header by default and needed to explicitly tell it to pass it through. Do you know whether relayd would pass the `Host` header to Synapse unaffected or modify/block it at all??

Reply:
I'm no expert either, but I ran a small test using the suggested config, and with netcat as the "server" that is forwarded to, to capture the request, and e.g. `curl https://example.com:18448/_matrix/hello` produces
```
GET /_matrix/hello HTTP/1.1
Accept: */*
Host: example.com:18448
User-Agent: curl/7.75.0
```
(port slightly altered in test config as otherwise I would get printouts from elsewhere trying to federate with my server)

And for the regular https port (`curl https://example.com`):
```
GET / HTTP/1.1
Accept: */*
Host: tlvb.se
User-Agent: curl/7.75.0
X-Forwarded-For: 256.92.34.25
X-Forwarded-Proto: https
```

So the host does not seem to be modified here.

Reply:
Indeed, that looks like what I would expect.